demisto_sdk/commands/common/tests/constants_regex_test.py

```diff
@@ -11,7 +11,7 @@
     PACKS_SCRIPT_TEST_PY_REGEX, PACKS_SCRIPT_YML_REGEX,
     PACKS_WIDGET_JSON_REGEX, PLAYBOOK_README_REGEX, PLAYBOOK_YML_REGEX,
     TEST_PLAYBOOK_YML_REGEX)
-from demisto_sdk.commands.common.tools import checked_type
+from demisto_sdk.commands.common.hook_validations.structure import checked_type_by_reg
```

Comment:
I think it makes sense to move this test to `structure_test`

Reply:
Moved it